firestore/tests/unit/test_transaction.py

```diff
@@ -837,7 +843,9 @@ def test_failure_second_attempt(self, _sleep):
         self.assertIs(exc_info.exception, exc2)
 
         # Verify mocks used.
-        _sleep.assert_called_once_with(1.0)
+        exponential_sleep_generator.assert_called_once_with(1.0,
```

Comment:
Please move `1.0,` to the next line (and indented).

https://www.python.org/dev/peps/pep-0008/#indentation